src/OrchardCore/OrchardCore.Abstractions/Modules/Services/ILocalClock.cs

```diff
@@ -5,10 +5,14 @@ namespace OrchardCore.Modules;
 /// </summary>
 public interface ILocalClock
 {
+    [Obsolete("This property has been deprecated, please use GetLocalNowAsync() instead.", error: true)]
+    Task<DateTimeOffset> LocalNowAsync => GetLocalNowAsync();
+
     /// <summary>
     /// Gets the time for the local time zone.
     /// </summary>
-    Task<DateTimeOffset> LocalNowAsync { get; }
+
+    Task<DateTimeOffset> GetLocalNowAsync();
```

Comment:
This introduces a breaking change. To avoid having a breaking change so this instead

```
Task<DateTimeOffset> LocalNowAsync { get; }

Task<DateTimeOffset> GetLocalNowAsync()
    => LocalNowAsync ;

```

Reply:
Why it's a breaking change? the value still comes from the new implementation, right?

Reply:
It's for when you have an existing alternative implementation. Should be rare but can happen: Then, with Mike's code, no change is needed, only when we remove `LocalNowAsync`.

Reply:
That's why I used error before to let the compiler complains whenever we use it

Reply:
> That's why I used error before to let the compiler complains whenever we use it

this will break people. So we have to avoid it. Doing my suggestion then no breaking change because in the interface we provide the default implementation. In your implementation, you can change the default behavior found in the interface.